python/modem-firmware-1.3+/README.md

```diff
@@ -321,6 +321,24 @@ file saved
 C:\utils>
 ```
 
+## nRF Cloud Device Provisioning
+The `nrf_cloud_provision.py` script performs device provisioning with nRF Cloud.
+Your nRF Cloud REST API key is a required parameter. See [https://nrfcloud.com/#/account](https://nrfcloud.com/#/account).
+Also required is a CSV file compatible with the [ProvisionDevice](https://api.nrfcloud.com/v1/#operation/ProvisionDevices) endpoint. You can use the provisioning CSV file produced by `device_credentials_installer.py`.
+
+```
+usage: nrf_cloud_provision.py [-h] --apikey APIKEY [--chk] [--csv CSV] [--res RES]
+
+nRF Cloud Device Provisioning
+
+optional arguments:
+  -h, --help       show this help message and exit
+  --apikey APIKEY  nRF Cloud API key (default: )
+  --chk            For single device provisioning, check if device exists before provisioning (default: False)
+  --csv CSV        Filepath to provisioning CSV file (default: provision.csv)
+  --res RES        Filepath where the CSV-formatted provisioning result(s) will be saved (default: )
+```
+
```

Comment:
i can... 
it just isn't that interesting and it is one more thing to maintain.

Reply:
I think it would be interesting if, say, it were a bulk process of 10 devices and one of them fails.  It would be good to see what such a failure looks like.  I do understand the burden of maintenance though.

Reply:
i suppose you are more interested in the output csv.
i will add those in a separate PR

Reply:
I thought saving the output csv is optional, and if not provided, you would print to the screen.  But sure, a file is fine instead.

Reply:
well yes, it just prints the CSV output if a file is not specified.
it doesnt look pretty.
it is meant to later inspect later via some other script if there are errors.